src/legacy/ui/public/vis/editors/default/components/default_editor_agg_select.tsx

```diff
@@ -62,9 +66,9 @@ function DefaultEditorAggSelect({
     />
   );
 
-  let aggHelpLink = null;
```

Comment:
I would rather suggest: `let aggHelpLink: string | null = null;`, since `null` feels like the more natural unset value here.